packages/editor/src/lib/editor/shapes/ShapeUtil.ts

```diff
@@ -65,7 +65,7 @@ export abstract class ShapeUtil<Shape extends TLUnknownShape = TLUnknownShape> {
 	 * @param shape - The shape.
 	 * @public
 	 */
-	abstract component(shape: Shape): any
+	abstract component(shape: Shape, isCulled: boolean): any
```

Comment:
Not sure I 100% like this. We have to do the `display: none` logic for culled shapes in all the shapes. OTOH it does the users the ability whether they want to cull shapes or not. Could make the perf slower for custom shapes if the users are not aware of this.

I guess we would need another div if we wanted to avoid this, which is also not the best?